api/db/migrate/20231005175407_add_hidden_in_reader_to_text_sections.rb

```diff
@@ -0,0 +1,5 @@
+class AddHiddenInReaderToTextSections < ActiveRecord::Migration[6.1]
+  def change
+    add_column :text_sections, :hidden_in_reader, :boolean, default: false
```

Comment:
This should have `null: false` as well as `default: false`. You can bring it down in your local DB by running:

```sh
bin/rake db:migrate:down VERSION=20231005175407
```